src/Interpreter.js

```diff
@@ -17,6 +17,7 @@ export default class Interpreter {
     programCounter: number;
     memory: { [string]: any };
     isRunning: boolean;
+    timeDelayMs: number;
```

Comment:
I think `stepTimeMs` is maybe a better name now that this is managed inside the Interpreter.